test/integration/manifests/cns/daemonset.yaml

```diff
@@ -131,4 +131,150 @@ spec:
+      annotations:
+        cluster-autoscaler.kubernetes.io/daemonset-pod: "true"
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: kubernetes.azure.com/cluster
+                operator: Exists
+              - key: type
+                operator: NotIn
+                values:
+                - virtual-kubelet
+              - key: beta.kubernetes.io/os
+                operator: In
+                values:
+                - windows
+      priorityClassName: system-node-critical
+      tolerations:
+        - key: CriticalAddonsOnly
+          operator: Exists
+        - operator: "Exists"
+          effect: NoExecute
+        - operator: "Exists"
+          effect: NoSchedule
+      containers:
+        - args:
+          - .\setkubeconfigpath.ps1
+          - ;
+          - .\azure-cns.exe
+          - -c
+          - tcp://$(CNSIpAddress):$(CNSPort)
+          - -t
+          - $(CNSLogTarget)
+          - -o
+          - $(CNSLogDir)
+          - -storefilepath
+          - $(CNSStoreFilePath)
+          - -config-path
+          - '%CONTAINER_SANDBOX_MOUNT_POINT%\$(CNS_CONFIGURATION_PATH)'
+          - --kubeconfig
+          - .\kubeconfig
+          command:
+            - powershell.exe
+          env:
+            - name: PATH
+              value: '%CONTAINER_SANDBOX_MOUNT_POINT%\Windows\System32\WindowsPowershell\v1.0\'
+            - name: CNSIpAddress
+              value: 127.0.0.1
+            - name: CNSPort
+              value: "10090"
+            - name: CNSLogTarget
+              value: stdoutfile
+            - name: CNSLogDir
+              value: /k/azurecns
+            - name: CNSStoreFilePath
+              value: /k/azurecns/
+            - name: CNS_CONFIGURATION_PATH
+              value: /etc/azure-cns/cns_config.json
+            - name: NODENAME
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: spec.nodeName
+          image: mcr.microsoft.com/containernetworking/azure-cns:v1.4.44.4
+          imagePullPolicy: IfNotPresent
+          name: cns-container
+          ports:
+            - containerPort: 10090
+              hostPort: 10090
+              name: api
+              protocol: TCP
+            - containerPort: 10092
+              hostPort: 10092
+              name: metrics
+              protocol: TCP
+          resources:
+            limits:
+              cpu: 250m
+              memory: 250Mi
+          securityContext:
+            privileged: true
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+          volumeMounts:
+            - mountPath: /k/azurecns
+              name: log
+            - mountPath: /etc/azure-cns
+              name: cns-config
+      initContainers:
+        - name: delete-azure-vnet-telemetry
```

Comment:
## What do we know?
- In Windows, you can rename a running process (binary)
- In Windows, you **cannot**, overwrite a running process (binary)
    -  It errors with "Access is denied"
    - This is why `cp` did not work in my powershell example (if you were there), but `mv` and `Rename-Item` did

## What does this mean for us?
- azure-vnet-telemetry.exe is a long-lived process, and we cannot overwrite the `azure-vnet-telemetry.exe.old` file, the 2nd time that `dropgz` runs
    - 1st time `dropgz` runs is fine (it can rename long-running `azure-vnet-telemetry.exe` to `azure-vnet-telemetry.exe.old` just fine
    - 2nd time however, `dropgz` tries to overwrite existing `azure-vnet-telemetry.exe.old`, but `azure-vnet-telemetry.exe.old` is still running as before (since it's long-lived)

## Solution
- We kill the `azure-vnet-telemetry.exe` process, before installing new `azure-vnet-telemetry.exe` binary through `dropgz`
- Then, since `dropgz` is an init container to `cns`, `cns` restarts and also restarts `azure-vnet-telemetry.exe`

Reply:
- `azure-vnet.exe` is not a problem, since we can rename binaries as they are running (that is the only case we would hit with `azure-vnet.exe`, since it is short-lived)
- The only possible issue with `azure-vnet.exe`, is if we happen to update `dropgz` **twice**, right in a row, **while** an `azure-vnet.exe` call is running in the background
     - This would error once, but then dropgz would retry and succeed the next call